functions/ingest/format/common.go

```diff
@@ -0,0 +1,48 @@
+package format
+
+import (
+	"github.com/nuclio/nuclio-sdk-go"
+	"github.com/v3io/v3io-tsdb/pkg/tsdb"
+	"github.com/v3io/v3io-tsdb/pkg/utils"
+	"sort"
+)
+
+var DEFAULT_FORMAT string = "DEFAULT"
+var TCOLLECTOR_FORMAT string = "TCOLLECTOR"
+
+type InputFormat interface  {
+	Ingest(tsdbAppender tsdb.Appender, event nuclio.Event) (interface{}, error)
```

Comment:
Usually in Go a single-method interface is named for its method (so `Ingester`, in this case). Also, this interface can easily be replaced by an if-else block 🙊

Reply:
I prefer to leave it as interface, it's nicer and might be extended in the future if needed. bare in mind that currently we have only 2 formats but we will have more